### PR TITLE
Wd nv 1558 11 remove conda section of readme

### LINKED ISSUE #11: remove conda section of README
ah we should probably remove this conda section

_Originally posted by @SamHollings in https://github.com/NHSDigital/RAP_example_pipeline_python/pull/6#discussion_r1280865220_

### LINKED ISSUE #12: get rid of "adapting for your project" section
We need to get rid of this whole "adapting for your project" section

_Originally posted by @SamHollings in https://github.com/NHSDigital/RAP_example_pipeline_python/pull/6#discussion_r1280868248_

### PR DESCRIPTION
**Pull request by warren-davies4**
Covers issues #11 and #12

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -203,56 +197,11 @@ This file currently runs a set of example steps using example data.
 
 This directory contains the meaty parts of the code. By organising the code into logical sections, we make it easier to understand, maintain and test. Moreover, tucking the complex code out of the way means that users don't need to understand everything about the code all at once.
 
-* `data_connections.py` handles reading data in and writing data back out.
+* `data_ingestion/get_data.py` handles reading data in
 * `processing` folder contains the core business logic.
 * `utils` folder contains useful reusable functions (e.g. to set up logging, and importing configuration settings from `config.toml`)
 * `write_excel.py` contains functions relating to the final part of the pipeline, any exporting or templating happens here. This is a simplistic application of writing output code to an Excel spreadsheet template (.xlsx). A good example of this application is: [NHS sickness absence rates publication](https://github.com/NHSDigital/absence-rates). We highly recommend to use [Automated Excel Production](https://nhsd-git.digital.nhs.uk/data-services/analytics-service/iuod/automated-excel-publications) for a more in depth Excel template production application.
```
Comment (SamHollings) [inline README.md]: change this line to talk about write_csv?

File: README.md [hunk 2]
```diff
@@ -150,28 +137,35 @@ To run the pipeline, enter the following command into the terminal:
...
 +---src                               <- Scripts with functions for use in 'create_publication.py'. Contains project's codebase.
 |   |       __init__.py               <- Makes the functions folder an importable Python module
 |   |
-|   +---utils                     <- Scripts relating to configuration and handling data connections e.g. importing data, writing to a database etc.
+|   +---utils                         <- Scripts relating to configuration and handling data connections e.g. importing data, writing to a database etc.
 |   |       __init__.py               <- Makes the functions folder an importable Python module
 |   |       file_paths.py             <- Configures file paths for the package
 |   |       logging_config.py         <- Configures logging
-|   |       data_connections.py       <- Handles data connections i.e. reading/writing dataframes from SQL Server
+|   |       spark.py                  <- For setting up and configuring Apache Spark
```
Comment (SamHollings) [inline README.md:151]: I wonder if we should have put the spark functions within the data_connections file...

File: README.md [hunk 3]
```diff
@@ -150,28 +137,35 @@ To run the pipeline, enter the following command into the terminal:
...
 |   | 
 |   +---processing                    <- Scripts with modules containing functions to process data i.e. clean and derive new fields
 |   |       __init__.py               <- Makes the functions folder an importable Python module
 |   |       clean.py                  <- Perform cleaning and wrangling processes 
-|   |       derive_fields.py          <- Create new field definitions, columns, derivations.
+|   |       aggregate_counts.py       <- Creates aggregations and counts based on the imported data
```
Comment (SamHollings) [inline README.md:156]: I wonder if we should have put the aggregate_counts functions within the dervied_fields file...

**Review by SamHollings — APPROVED** [review]
Some small comments, musings really.

**Review by SamHollings — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (warren-davies4) [reply README.md]: good point! I've updated this

File: README.md (hunk 2 above)
Comment (warren-davies4) [reply README.md:151]: The spark.py file currently only has creating the spark session at present, not really a data connection exactly. In the data_ingestion folder there is reading_data.py which creates the dataframe from the csv

File: README.md (hunk 3 above)
Comment (warren-davies4) [reply README.md:156]: I feel like aggregate_counts is more direct as to what's happening in there. When I think of derive fields, I think more like adding new columns to the existing data, like creating age band from an age or something.

But I guess it's six of one, half a dozen of the other! happy to go with derive_fields if you prefer that.

File: README.md (hunk 1 above)
Comment (warren-davies4) [reply README.md]: do we need the Excel parts at all for this? Should I remove the templates folder and references to it also?

**Review by SamHollings — APPROVED** [review]

**Pull request merged**